demisto_sdk/commands/common/constants.py

```diff
@@ -225,6 +225,14 @@ def no_common_server_python(path):
     def invalid_file_path(file_path):
         return f"Found incompatible file path: {file_path}."
 
+    @staticmethod
+    def invalid_v2_integration_name(file_path):
+        return f"The name of the v2 integration : {file_path} is incorrect , should be *name* v2."
```

Comment:
please print the actual integration display name

Reply:
maybe:
"Current display name is: {display}
change display name to be of the format: *name* v2
e.g: Kenna v2"